SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Ingestion/Http/IngestionHttp.cs

```diff
@@ -98,41 +64,24 @@ public IServiceCall PrepareServiceCall(string appSecret, Guid installId, IList<L
             return new HttpServiceCall(this, logs, appSecret, installId);
         }
 
-        internal HttpRequestMessage CreateRequest(string appSecret, Guid installId, string requestContent)
+        internal IDictionary<string, string> CreateHeaders(string appSecret, Guid installId)
         {
-            var baseUrl = string.IsNullOrEmpty(_baseLogUrl) ? DefaultBaseUrl : _baseLogUrl;
-
-            // Create HTTP transport objects
-            var request = new HttpRequestMessage
+            // Log headers.
+            MobileCenterLog.Verbose(MobileCenterLog.LogTag, "Headers: " +
+                    $"{AppSecret}={GetRedactedAppSecret(appSecret)}, " +
+                    $"{InstallId}={installId}");
+            return new Dictionary<string, string>
             {
-                Method = HttpMethod.Post,
-                RequestUri = new Uri(baseUrl + ApiVersion)
+                { AppSecret, appSecret },
+                { InstallId, installId.ToString() }
             };
-            MobileCenterLog.Verbose(MobileCenterLog.LogTag, $"Calling {request.RequestUri}...");
-
-            // Set Headers
-            request.Headers.Add(AppSecret, appSecret);
-            request.Headers.Add(InstallId, installId.ToString());
-
-            // Log headers
-            var headers = $"Headers: Content-Type={ContentTypeValue}, " +
-                            $"{AppSecret}={GetRedactedAppSecret(appSecret)}, " +
-                            $"{InstallId}={installId}";
-            MobileCenterLog.Verbose(MobileCenterLog.LogTag, headers);
-
-            // Request content
-            MobileCenterLog.Verbose(MobileCenterLog.LogTag, requestContent);
```

Comment:
The request content is no longer printed after the change, can you please add this back?